rllib/utils/tests/test_actor_manager.py

```diff
@@ -0,0 +1,152 @@
+import random
+import sys
+import time
+import unittest
+
+import ray
+from ray.experimental.state.api import list_actors
+from ray.rllib.utils.actor_manager import FaultTolerantActorManager
+
+
+@ray.remote(max_restarts=-1)
+class Actor:
+    def __init__(self, i):
+        random.seed(i)
+        self.count = 0
+
+    def call(self):
+        self.count += 1
+        r = random.random()
+
+        # 10% chance of crashing.
+        if r < 0.1:
+            sys.exit(1)
+        # Another 10% chance of throwing errors.
+        elif r < 0.2:
+            raise AttributeError("sorry")
+
+        # Otherwise, return good result.
+        return self.count
+
+    def apply(self, func):
+        return func(self)
+
+
+def wait_for_restore():
```

Comment:
Can add a doctsring on what this method does?  highlevel

Reply:
done